test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -eu
 
 echo; echo "Creating ServiceAccount..."
-kubectl create sa test-user
+kubectl create sa test-user --dry-run -o yaml | kubectl apply -f -
```

Comment:
The `--dry-run` option has been deprecated since v1.18+, so could you replace them with `--dry-run=client`?

Reply:
I thought I saw some deprecation error...
Fixed them.
Thank you.